part-2.py

```diff
@@ -4,13 +4,33 @@
 # the appropriate comment.
 
 # search
-
+def search(array, query):
```

Comment:
👍 Just a note that `array[1:]` makes a copy of the array element-by-element.  This means this method is O(n^2) in time/space complexity.